cratonclient/tests/unit/test_main_shell.py

```diff
@@ -0,0 +1,25 @@
+#   Licensed under the Apache License, Version 2.0 (the "License"); you may
+#   not use this file except in compliance with the License. You may obtain
+#   a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#   WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#   License for the specific language governing permissions and limitations
+#   under the License.
+
+"""Tests for `cratonclient.shell.main` module."""
+
+from cratonclient.tests import base
+from cratonclient.shell import main
+
+
+class TestShell(base.TestCase):
+    '''Test our craton main shell.'''
+
+    def test_return(self):
+        '''Verify that cratonclient shell main returns as expected.'''
+        print("Craton Shell Help Test")
```

Comment:
This is NOT the Help Test.  It's just the basic test that will be removed once real functionality is added to the main shell.  Change print or remove